core/src/test/java/com/graphhopper/routing/util/parsers/FootTagParserTest.java

```diff
@@ -63,7 +61,7 @@ public FootTagParserTest() {
 
     @Test
     public void testGetSpeed() {
-        EdgeIntAccess edgeIntAccess = new ArrayEdgeIntAccess(encodingManager.getIntsForFlags());
+        EdgeIntAccess edgeIntAccess = new ArrayEdgeIntAccess(encodingManager.getBytesForFlags() / 4 + 1);
```

Comment:
Given how frequently we seem to use this we could just keep `getIntsForFlags` next to `getBytesForFlags` for such cases where need to count full ints?

Reply:
Ok, I just thought it wouldn't be wrong to ask the encoding manager how many ints it needs even though it uses bytes internally. 

> We could create a TestArrayEdgeIntAccess 

Or a static creator using bytes for ArrayEdgeIntAccess.